db/db.go

```diff
@@ -1053,18 +1053,23 @@ func saveValidators(data *types.EpochData, tx *sqlx.Tx, client rpc.Client) error
 	}
 
 	var currentState []*types.Validator
-	err := tx.Select(&currentState, "SELECT validatorindex, withdrawableepoch, withdrawalcredentials, slashed, activationeligibilityepoch, activationepoch, exitepoch, lastattestationslot, status FROM validators;")
+	err := tx.Select(&currentState, "SELECT validatorindex, withdrawableepoch, withdrawalcredentials, slashed, activationeligibilityepoch, activationepoch, exitepoch, status FROM validators;")
```

Comment:
Now that `lastattestationslot` is not part of the table anymore, consider removing the `db` tag and changing the datatype here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/36a95eb96c69036c42bc6891d98cd5d177b592af/types/exporter.go#L87

Reply:
that needs to be done in a separate bids, as the migration script still uses that mapping

Reply:
Right :facepalm: